modulemd/v2/include/modulemd-2.0/modulemd-module-stream.h

```diff
@@ -240,8 +242,6 @@ modulemd_module_stream_upgrade (ModulemdModuleStream *self,
 
 
 /**
- * modulemd_module_stream_validate:
- * @self: (in): This #ModulemdModuleStream.
```

Comment:
Why are you deleting these lines? I'm guessing this was a mistake. It will break the documentation-generation for this function.